src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -4,6 +4,8 @@
     <ProjectGuid>{89F37791-6254-4D60-AB96-ACD3CCA0E771}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Configurations>netcoreapp-OSX-Debug;netcoreapp-OSX-Release;netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
+    <!-- This is needed so that code for TlsCipherSuite will have no namespace (causes compile errors) when used within T4 template  -->
+    <DefineConstants>PRODUCT</DefineConstants>
```

Comment:
We usually also assume there might existing constants, so this should be updated to be:

```
<DefineConstants>$(DefineConstants);PRODUCT</DefineConstants>
```